src/test/java/org/openelisglobal/patient/PatientServiceTest.java

```diff
@@ -97,413 +84,6 @@ public void getData_shouldCopyPropertiesFromDatabase() throws Exception {
         Assert.assertEquals(gender, savedPatient.getGender());
     }
 
-    @Test
```

Comment:
@josephbate , why did you have to delete the existing tests ??

Reply:
my local repository got corrupted and i didn't noticed it till when i pushed it 
but i have restored it

Reply:
you never got rid of the chanes here

Reply:
can you discard the changes here